src/pseudo/NormConservingPsp.jl

```diff
@@ -77,7 +77,9 @@ potential:
 lim{q->0} 4π Nelec ∫{R+} (V(r) - C(r)) sin(qr)/qr r^2 dr + F[C(r)]
 = 4π Nelec ∫{R+} (V(r) + Z/r) r^2 dr
 """
-eval_psp_energy_correction(T, psp::NormConservingPsp, n_electrons) = zero(T)
+function eval_psp_energy_correction(T, psp::NormConservingPsp, n_electrons)
+    error("`eval_psp_energy_correction not implemented for $(typeof(psp))")
+end
```

Comment:
Sorry, but this I veto. If we don't want to implement a method for this function, then just don't implement a method and just define the function (i.e. `function eval_psp_energy_correction end`) and let the compiler throw the error at compile time.